packages/navigation/src/styles/Navigation.module.css

```diff
@@ -57,8 +57,7 @@
   align-items: center;
```

Comment:
Lots of changes in this file, the gist is:

* The tabs navigation are going from 24px left/right padding down to `15`.
* The bottom border is moving from the `a` to the `span` that has the navigation item text, so that it more closely hugs the text width without reducing click target area

Reply:
Why 15 instead of the 8-based 16?

Reply:
We talked through this, landed on `12px` for left/right padding in the `a` element, `4` for left/right padding in the `span` for the tab label (`16px` in total on either side)